src/gui/guiChatConsole.cpp

```diff
@@ -43,6 +43,11 @@ inline bool isInCtrlKeys(const irr::EKEY_CODE& kc)
 	return kc == KEY_LCONTROL || kc == KEY_RCONTROL || kc == KEY_CONTROL;
 }
 
+inline u32 getScrollbarSize(IGUIEnvironment* env)
+{
+	return env->getSkin() ? env->getSkin()->getSize(gui::EGDS_SCROLLBAR_SIZE) : 16;
```

Comment:
this 16 must be a named constant

Reply:
actually you should be able to remove that check, Minetest always initializes an IGUISkin

I know there's a lot of checks like this elsewhere (we should get rid of them), but there are also places where the IGUISkin is used unconditionally